unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/WebInterfacePassportApiBridge.cs

```diff
@@ -29,5 +36,14 @@ await DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(u
 
             return nftList;
         }
+
+        public async UniTask<Nft> QueryNftCollectionMatic(string userId, string urn)
```

Comment:
also needs cancellation suppot.

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/4006/commits/b2664bebab0dbfe34c26f10fa95feba65e77b41b. Thanks!!